environment.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+############################################################
+# Help                                                     #
+############################################################
+Help()
+{
+   # Display Help
+   echo "Add description of the script functions here."
+   echo
+   echo "Syntax: scriptTemplate [-h|t|v]"
+   echo "options:"
+   echo "h     Print this Help."
+   echo "v     Verbose mode."
+   echo "t     Type of conda environment. Either simulation (default) or artifact."
+}
+
+# Define variables
+env_type="simulation"
+
+# Process input options
+while getopts ":ht:" option; do
+   case $option in
+      h) # display help
+         Help
+         return;;
+      t) # Type of conda environment to build
+         env_type=$OPTARG;;
+     \?) # Invalid option
+         echo "Error: Invalid option"
+         return;;
+   esac
+done
+
+# Parse environment name
+env_name=$(basename "`pwd`")
+env_name+="_$env_type"
+
+# Check if environment exists already
+env_exists=$(conda info --envs | grep $env_name | head -n 1)
+if [[ $env_exists == '' ]]; then
+  env_exists="no"
+else
+  env_exists="yes"
+fi
+
+if [ $env_exists == 'yes' ]; then
+  echo "Existing environment found for $env_name."
+fi
+
+# If environment does not exit, create the new environment
+if [ $env_exists == 'no' ]; then
```

Comment:
Nit: this can be combined into an if-else black:

```bash
if [ $env_exists == 'yes' ]; then
  echo "Existing environment found for $env_name."
else
  # If environment does not exit, create the new environment
  ...
fi
```